Source/WebCore/platform/graphics/wpe/PlatformDisplayWPE.cpp

```diff
@@ -33,9 +33,25 @@
 
 namespace WebCore {
 
+static std::pair<const uint8_t*, size_t>* s_initializationData = nullptr;
+
+void PlatformDisplayWPE::initialize(std::pair<const uint8_t*, size_t> data)
+{
+    auto initializationData = std::make_unique<std::pair<const uint8_t*, size_t>>(data);
+    s_initializationData = initializationData.get();
```

Comment:
We need to find a better (or rather more elegant) way to handle this. Works for now, but it's a hack.